scripts/flying-tokens.js

```diff
@@ -40,6 +49,39 @@ Hooks.on('renderTokenHUD', (app, html, data) => {
         FlyingHud.renderHud(app, html, data);
 });
 
+Hooks.on("renderTokenConfig", (app, html, data) => {
+    if (isFlyer(app.token)) {
+        let altToken = app.token.getFlag(MODULE, "altToken") || "";
+        let newHtml = `<div class="form-group">
+                    <label>Flying Image Path</label>
+                    <div class="form-fields">
+                      <button type="button" class="file-picker" data-type="imagevideo" data-target="flags.${MODULE}.altToken" title="Browse Files" tabindex="-1">
+                        <i class="fas fa-file-import fa-fw"></i>
+                      </button>
+                      <input class="image" type="text" name="flags.${MODULE}.altToken" placeholder="path/image.png" value="${altToken}">
+                    </div>
+                  </div>`
+        const tinthtml = html.find('input[name="texture.src"]');
+        const formGroup = tinthtml.closest(".form-group");
+        formGroup.after(newHtml);
+        // altToken = html.find('input[name="flags.flyingTokens.alt"]').value;
+        app.setPosition({ height: "auto" });
+        html.find(`button[data-target="flags.${MODULE}.altToken"]`).on('click', async () => {
+            new FilePicker({
+                current: altToken,
+                type: "imagevideo",
+                displayMode: "tiles",
+                button: "file-picker",
```

Comment:
this `button` property I don't know if this is really necessary

Reply:
If it works without it then it's not needed

Reply:
lol it does work